lib/apiservers/portlayer/restapi/configure_port_layer.go

```diff
@@ -85,6 +85,18 @@ func configureAPI(api *operations.PortLayerAPI) http.Handler {
 		log.Fatalf("configure_port_layer ERROR: %s", err)
 	}
 
+	// Configure the func invoked if the PL panics or is restarted by vic-init
+	api.ServerShutdown = func() {
+		log.Infof("Shutting down port-layer-server")
+
+		if sess != nil {
```

Comment:
I think it is redundant. Session can not be nil if Create doesn't return an error.